paper-tooltip.html

```diff
@@ -70,10 +70,46 @@
 
       @apply(--paper-tooltip);
     }
+
+    #arrow {
```

Comment:
This is not part of the Material Design spec, and it was omitted on purpose, I'm afraid.